File: 26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 0
        for i in range(k+1, len(nums)):
            if nums[i]!=nums[k]:
                nums[k+1] = nums[i]
                k = k +1
        return k+1

File: 26-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int l = 0;
        for(int r=l+1; r<nums.size(); ++r){
            if(nums[r]!=nums[l]){
                nums[l+1]=nums[r];
                l++;
            }
        }
        return l+1;
    }
};